db/migrate/20160523202611_create_profiles.rb

```diff
@@ -0,0 +1,13 @@
+class CreateProfiles < ActiveRecord::Migration
+  def change
+    create_table :profiles do |t|
+
+      t.integer :user_id, index: true
+      t.string  :user_full_name
+      t.string  :user_email
+      t.string  :user_phone
+      t.datetime :created_at, null: false
+      t.datetime :updated_at, null: false
```

Comment:
`created_at` and `updated_at` can be changed to `timestamps`(in 1 line instead of 2)